src/os/style/style.js

```diff
@@ -57,6 +57,60 @@ os.style.DEFAULT_FEATURE_SIZE = 3;
 os.style.DEFAULT_STROKE_WIDTH = 3;
 
 
+/**
+ * The default style fields to check for line dashes.
+ * @type {!Array<string>}
+ * @const
+ */
+os.style.DEFAULT_LINE_DASH_STYLE_FIELDS = [
+  os.style.StyleField.STROKE
+];
+
+
+/**
+ * Default line style for geometries. Specifically for lines/polygons.
+ * @type {!Array<number>}
+ */
+os.style.DEFAULT_LINE_STYLE = [];
```

Comment:
The default `lineDash` should be `undefined`, so this constant shouldn't be needed at all.

If a line is not dashed, all configs/styles/etc should be exactly as they were before this PR and there shouldn't be any additional processing required, other than the property check.